test/python/quantum_info/states/test_statevector.py

```diff
@@ -106,73 +106,85 @@ def test_init_statevector(self):
 
     def test_from_circuit(self):
         """Test initialization from a circuit."""
-        # random unitaries
-        u0 = random_unitary(2).data
-        u1 = random_unitary(2).data
-        # add to circuit
-        qr = QuantumRegister(2)
-        circ = QuantumCircuit(qr)
-        circ.unitary(u0, [qr[0]])
-        circ.unitary(u1, [qr[1]])
-        target = Statevector(np.kron(u1, u0).dot([1, 0, 0, 0]))
-        vec = Statevector.from_instruction(circ)
-        self.assertEqual(vec, target)
-
-        # Test tensor product of 1-qubit gates
-        circuit = QuantumCircuit(3)
-        circuit.h(0)
-        circuit.x(1)
-        circuit.ry(np.pi / 2, 2)
-        target = Statevector.from_label("000").evolve(Operator(circuit))
-        psi = Statevector.from_instruction(circuit)
-        self.assertEqual(psi, target)
-
-        # Test decomposition of Controlled-Phase gate
-        lam = np.pi / 4
+        # # random unitaries
+        # u0 = random_unitary(2).data
+        # u1 = random_unitary(2).data
+        # # add to circuit
```

Comment:
Looks like something's gone wrong in your commits here.

Reply:
Yes indeed! Should be fixed now 🙂